src/util/worker.ts

```diff
@@ -403,7 +403,6 @@ export async function runWorkers(
 ) {
   logger.info(`Creating ${numWorkers} workers`, {}, "worker");
 
-  const workers = [];
```

Comment:
Nice catch

Reply:
yep, already quickfix merged this into main before as well.